adoc/chapters/programming_interface.adoc

```diff
@@ -1800,8 +1800,14 @@ info::device::max_work_group_size
 ----
 
     @ [.code]#size_t#
-   a@ Returns the maximum number of work-items that are permitted in a work-group executing a kernel on a single compute unit.
+   a@ Returns the maximum number of work-items in a work-group that this device
+      is capable of executing on a single compute unit.
```

Comment:
I think the mention of a "single compute unit" doesn't add value here.  The whole notion of a compute unit is a little fuzzy anyway, and the wording here implies that there might be a different number of work-items if a work-group executed over a different number of compute units.

```suggestion
   a@ Returns the maximum number of work-items that this device
      is capable of executing in a work-group .
```

Reply:
Sounds good to me.  @bashbaug, I saw that you gave this a thumbs-up.  Note that [OpenCL](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#CL_DEVICE_MAX_WORK_GROUP_SIZE) has the same wording.  Should that be changed also?